src/main/scala/org/broadinstitute/dsde/workbench/sam/service/StatusService.scala

```diff
@@ -35,7 +36,7 @@ class StatusService(
 
   private def checkOpenDJ(groupToLoad: WorkbenchGroupName): IO[SubsystemStatus] = {
     logger.info("checking opendj connection")
-    directoryDAO.loadGroupEmail(groupToLoad).map {
+    directoryDAO.loadGroupEmail(groupToLoad, SamRequestContext(null)).map {
```

Comment:
this is used from `healthMonitor` on line 29. Any issues with doing it like this?

Reply:
this is fine